documentation/troubleshooting.md

```diff
@@ -251,3 +251,11 @@ To change your script execution policy:
     documentation about [execution
     policies](http://go.microsoft.com/fwlink/?LinkID=135170) and [how to set
     them](https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.security/set-executionpolicy).
+
+## Getting error 'Could not parse PKey: no start line' using SSH private key
+
+This error indicates that the SSH private key provided is encrypted, which is unsupported by Bolt
+if the key is provided under `key-data`. You can manually decrypt the key by running `openssl rsa -in
+<key_file>` and providing your passphrase, if providing a decrypted key is feasible for your use
+case and security practices. You can also add the key to your SSH agent and *not* specify a
+`private-key` for Bolt to use, and Bolt will use the agent to authenticate your connection.
```

Comment:
```suggestion
## 'Could not parse PKey: no start line' error message when using SSH private key

Bolt does not support encrypted SSH private keys if the keys are provided using the
`key-data` field in your transport configuration. If providing a decrypted key is feasible
for your use case and security practices, you can manually decrypt the key by running
`openssl rsa -in <KEY FILE>` and providing your passphrase. Alternatively, you can
add the key to your SSH agent and *not* specify a `private-key` for Bolt to use. Bolt
will use the agent to authenticate your connection.
```

Reply:
@lucywyman Could you move the "I still need help" section down? That should be the last section on the page.